frontend/services/image_presentation/ImagePresentation_Sinks.hpp

```diff
@@ -6,12 +6,14 @@
  */
 
 #include "ImageWrapper.h"
+#include <glowl/GLSLProgram.hpp>
 
 namespace megamol::frontend {
 
     struct glfw_window_blit {
         unsigned int gl_fbo_handle = 0;
         unsigned int fbo_width = 0, fbo_height = 0;
+        std::shared_ptr<glowl::GLSLProgram> blit_shader;
```

Comment:
To avoid pulling GL headers into the Image Presentation Service i would try to hide the glowl headers and shader class. Maybe via pimpl or so?